org.freedesktop.gitlab.rncbc.qpwgraph.yml

```diff
@@ -0,0 +1,24 @@
+app-id: org.freedesktop.gitlab.rncbc.qpwgraph
```

Comment:
this is the wrong app-id:

See
https://gitlab.freedesktop.org/rncbc/qpwgraph/-/tree/main/src/appdata

Reply:
Thanks!